backend/Fields/BatchPostFields.ts

```diff
@@ -23,7 +23,7 @@ router.post<Params, ResponseBody, RequestBody>(
 
     const { nounId } = req.params;
 
-    let noun: NounAttributes;
+    let noun: NounModel | null;
```

Comment:
Did you mean to include a single pipe in the type declaration here?

Reply:
Yeah I'm typescript the single pipe is used to mean "or"